frontend/packages/console-shared/src/components/dashboard/utilization-card/TopConsumerPopover.tsx

```diff
@@ -263,10 +263,9 @@ export const PopoverBody = withDashboardResources<DashboardItemProps & PopoverBo
               : t('console-shared~Top consumers')}
           </div>
           {consumers.length > 1 && (
-            <Dropdown
+            <ConsoleSelect
```

Comment:
switching between top consumer measure type will close the modal

https://github.com/user-attachments/assets/6d6f769d-3aa4-4044-a68d-70d316f59f94

Reply:
fixed. thanks!